skaha/opencadc.gradle

```diff
@@ -1,21 +1,7 @@
 configurations {
-    checkstyleDep
-    intTestImplementation.extendsFrom testImplementation
```

Comment:
The `intTestImplementation` still needs to be there to run the integration tests.